docs/using.rst

```diff
@@ -415,9 +422,13 @@ to, `install Docker`_, then issue the following command:
                -v "/var/lib/letsencrypt:/var/lib/letsencrypt" \
                quay.io/letsencrypt/letsencrypt:latest auth
 
-and follow the instructions (note that ``auth`` command is explicitly
-used - no installer plugins involved). Your new cert will be available
-in ``/etc/letsencrypt/live`` on the host.
+Certbot will obtain a certificate and place it in the directory
+``/etc/letsencrypt/live`` on your system and display further instructions 
+for installing the certificates. You must use the ``auth`` command
+to install the certificates instead of plug-ins for this method. 
```

Comment:
Few thoughts on this section.

1. We don't actually display any information about how to install the certificates. Doing so is extremely webserver specific. We have a bit of that information under [where are my certificates](https://certbot.eff.org/docs/using.html#where-are-my-certificates).
2. `auth` doesn't install certificates, it simply obtains them. Actually, `auth` is simply an alias for the `certonly`  command which we switched to after talking to some usability researchers (but kept the old command for backwards compatibility). We should actually change references of `auth` here to `certonly`. We can do this in a separate PR if you prefer.
3. I think we should be consistent with our use of plugins vs plug-ins. We normally use plugins, but we could potentially switch if you think there's value in doing so.
4. Certbot still actually uses the plugin system here. It just doesn't use any installer plugins. Take a look at our [plugin documentation](https://certbot.eff.org/docs/using.html#plugins) to see the distinction.